### ISSUE TITLE
Export Config.Logger

### ISSUE THREAD

**Issue by peter-edge** [yarpc/yarpc-go#894]
We would benefit from logging in Crossdock and in examples.

[later edit from @akshayjshah]: We currently have some logging support in YARPC, but the configuration field for users to supply a logger is unexported. This means that, in practice, we don't log. Current plan for this is in T841919. The short answer is that yes, we can do this quite soon.

**Comment by breerly** [comment]
Support a logger where? Can you rephrase this title and description to be more... descriptive?

Ideally there would be a background and/or motivation, plus a detailed title.

**Comment by akshayjshah** [comment]
I'll amend the title again - I think a little context got lost between multiple editors. Short answer: yes.

**Comment by peter-edge** [comment]
@akshayjshah ping haha :)

**Comment by akshayjshah** [comment]
Landed.
